src/commands/openWorkflowStepLogs.ts

```diff
@@ -0,0 +1,22 @@
+import * as vscode from "vscode";
+import {WorkflowStepNode} from "../treeViews/workflows/workflowStepNode";
+
+type WorkflowStepCommandArgs = Pick<WorkflowStepNode, "job" | "step" | "gitHubRepoContext">;
+
+export function registerOpenWorkflowStepLogs(context: vscode.ExtensionContext) {
+  context.subscriptions.push(
+    vscode.commands.registerCommand("github-actions.step.logs", async (args: WorkflowStepCommandArgs) => {
+      const job = args.job.job;
+      let url = job.html_url ?? "";
+      const stepName = args.step.name;
+
+      const index = job.steps && job.steps.findIndex(step => step.name === stepName) + 1;
+
+      if (url && index) {
+        url = url + "#step:" + index.toString() + ":1";
```

Comment:
If there are 4 steps in a run, you can navigate to each one like so (note `#step:` at the end of the URL):

`https://github.com/bbq-beets/ketchuponmyketchup/actions/runs/4810296620/jobs/8562724204#step:1:1`
`https://github.com/bbq-beets/ketchuponmyketchup/actions/runs/4810296620/jobs/8562724204#step:2:1`
`https://github.com/bbq-beets/ketchuponmyketchup/actions/runs/4810296620/jobs/8562724204#step:3:1`
`https://github.com/bbq-beets/ketchuponmyketchup/actions/runs/4810296620/jobs/8562724204#step:4:1`